XamlControlsGallery/XamlControlsGallery.sln

```diff
@@ -1,7 +1,7 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 15
-VisualStudioVersion = 15.0.27130.2027
+# Visual Studio Version 16
+VisualStudioVersion = 16.0.28714.193
```

Comment:
Please revert changes to this file.

Reply:
Fixed!

Reply:
Related commit: https://github.com/Microsoft/Xaml-Controls-Gallery/pull/73/commits/6c13455247f0c10cce518bf65fe2cd98e341c7c2